eclair-core/src/main/scala/fr/acinq/eclair/io/Peer.scala

```diff
@@ -44,15 +44,10 @@ import scodec.bits.ByteVector
  *
  * Created by PM on 26/08/2016.
  */
-class Peer(val nodeParams: NodeParams, remoteNodeId: PublicKey, switchboard: ActorRef, router: ActorRef, watcher: ActorRef, relayer: ActorRef, wallet: EclairWallet) extends FSMDiagnosticActorLogging[Peer.State, Peer.Data] {
+class Peer(val nodeParams: NodeParams, remoteNodeId: PublicKey, watcher: ActorRef, relayer: ActorRef, wallet: EclairWallet) extends FSMDiagnosticActorLogging[Peer.State, Peer.Data] {
 
   import Peer._
 
-  val reconnectionTask: ActorRef = context.actorOf(ReconnectionTask.props(nodeParams, remoteNodeId, switchboard, router), "reconnection-task")
-  // we register the reconnection task to our transitions ourselves; if we let the child actor register itself, there is
```

Comment:
What happened to that comment? Why isn't it an issue anymore?

Reply:
This isn't an issue anymore, because `Peer` actively sends the event on transition, instead of the child asynchronously registering to state transitions.